Core/Object Arts/Dolphin/System/Announcements/Announcement.cls

```diff
@@ -1,29 +1,29 @@
-"Filed out from Dolphin Smalltalk 7"!
-
-Object subclass: #Announcement
-	instanceVariableNames: ''
-	classVariableNames: ''
-	poolDictionaries: ''
-	classInstanceVariableNames: ''!
-Announcement guid: (GUID fromString: '{12525b20-16ef-4684-b01e-956b227a429d}')!
-Announcement comment: 'This class is the superclass for events that someone might want to announce, such as a button click or an attribute change. Typically you create subclasses for your own events you want to announce.

 
'!
-!Announcement categoriesForClass!Announcements-Core! !
-!Announcement methodsFor!
-
-asAnnouncement
	^ self! !
-!Announcement categoriesFor: #asAnnouncement!converting!public! !
-
-!Announcement class methodsFor!
-
-, anAnnouncementClass
	^ AnnouncementSet with: self with: anAnnouncementClass!
-
-asAnnouncement
	^ self new!
-
-handlesAnnouncement: anAnnouncement
	"The receiver acts as a filter to determine whether subscribers who used the receiver as signaling tag (event identifier class or symbol) should receive incoming announcement. In particular, registering to a superclass will receive the announcements from all subclasses."


	^ anAnnouncement class == self or: [ anAnnouncement class inheritsFrom: self ]!
-
-systemIcon

	^ Smalltalk ui icons iconNamed: #announcementIcon! !
-!Announcement class categoriesFor: #,!public! !
-!Announcement class categoriesFor: #asAnnouncement!converting!public! !
-!Announcement class categoriesFor: #handlesAnnouncement:!public!testing! !
-!Announcement class categoriesFor: #systemIcon!*Polymorph/Widgets/Themes!public! !
-
+"Filed out from Dolphin Smalltalk 7"!
+
+Object subclass: #Announcement
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+Announcement guid: (GUID fromString: '{12525b20-16ef-4684-b01e-956b227a429d}')!
+Announcement comment: 'This class is the superclass for events that someone might want to announce, such as a button click or an attribute change. Typically you create subclasses for your own events you want to announce.

 
'!

```

Comment:
There are bad line endings in all these Announcement related classes. I think it must have been filed in from Squeak/Pharo before the support was added for correctly reading mixed line endings in file-ins. I'll raise an issue for this.